miqi/kun_runtime/tool_host.py

```
@@ -255,10 +257,21 @@ async def execute(
             if (
                 collab_verdict == CollabVerdict.CONFIRM
                 and context.await_user_input is not None
-                # Reasoning mode (issue #680): fast = 信息型操作直接执行，
-                # 不弹确认卡（用户：极速模式完全不可能快）。权限型仍由
-                # ExecutionPolicy/approval 门控制。
-                and context.mode != "fast"
+                and context.mode == "fast"
+                # #741/#680 审阅 P1: fast 只跳过**信息型**（READ）工具的确认
```

Ruff (0.16.1)

[warning] Comment contains ambiguous `（` (FULLWIDTH LEFT PARENTHESIS). Did you mean `(` (LEFT PARENTHESIS)?

(RUF003)